### DISCUSSION TITLE
[future request]  add pop-up status indicate file synced to remote

### DISCUSSION THREAD

**Discussion by lovitus** [iterate-ch/cyberduck#15887] [Q&A]
**Is your feature request related to a problem? Please describe.**
    
- Every time I modify a file using Cyberduck's "Open With" feature in VSCode, I have to manually check Cyberduck after every Ctrl+S to confirm if the upload was successful or if a network issue caused a failure.

-  Yes, there have been instances where Cyberduck truncated my files to 0-byte empty files.

![image](https://github.com/iterate-ch/cyberduck/assets/14361541/2d98a45a-f191-41a9-b3c9-0ac62b97f1a1)

**Describe the solution you'd like**
    
- Add an option in Preferences to enable "Show Pop-Up Syncing Status."
- It would be good to receive a notification on macOS, and for windows a temporary pop-up info in the bottom right corner.

**Describe alternatives you've considered**
- Perhaps the Cyberduck developers will find some better solutions.

**Additional context**
Thank you for this wonderful app. Have a nice day every day!

**Comment by dkocher** [comment]
You should enable _Notifications_ [^1] in system settings.

[^1]: https://docs.cyberduck.io/cyberduck/notifications/

**Reply by lovitus** [reply]
Thank you, dkocher. Yes, for file uploads/downloads, there is a "transfer" dialog that shows the status and number.

However, if you use "Open With" to modify remote files, after you save them, there is no information to help you confirm if they have synced to the remote. You have to open the Cyberduck main UI to check the status bar (bottom area) or verify the file modification time.
